lib/xcpretty/term.rb

```diff
@@ -0,0 +1,26 @@
+
+module XCPretty
+  class Term
+
+    COLOR_DETECTION_COMMAND = "tput colors"
+    LOCALE_DETECTION_COMMAND = "locale charmap"
+    UNICODE_LOCALE = "UTF-8"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.